packages/utils/src/conversions.ts

```diff
@@ -5,7 +5,7 @@
  * conversions involving bigints, hexadecimals and buffers.
  * The module is structured with clear function naming to indicate
  * the conversion direction (e.g., `bigIntToHexadecimal` for BigInt
- * to hexadecimal, `bufferToBigint` for buffer to bigint) and employs
+ * to hexadecimal, `bufferToBigInt` for buffer to bigint) and employs
```

Comment:
I somehow missed this module documentation on my first review.  I think I'm used to skipping over big comment blocks at the tops of files which are usually copyright text.

Regardless, I like this description - very clear.

Reply:
I also skit it usually. It will be more visible on the TypeDoc website I guess.